x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/utils/helpers.test.tsx

```diff
@@ -1780,6 +1780,11 @@ describe('Exception helpers', () => {
               overrideField: 'agent.status',
               label: 'Agent status',
             },
+            {
```

Comment:
Focusing on this test file because it's the one thing in this PR owned by Detection Engine:

I don't think this test is doing anything useful right now. The test is described in a way that I would expect it to fail if any of these values were found in `filteredFields`, but that's not how it works. It's effectively `expect(result).not.toEqual(expect.arrayContaining([]))`, which is ultimately just an inequality comparison _on the entire array_.

Adding this new object to `arrayContaining` is actually going to _loosen_ this test and make it less likely to fail, as shown in the following example:

```ts
// pass
expect([{ foo: 'bar' }]).not.toEqual(expect.arrayContaining([{}]));
// fail
expect([{ foo: 'bar' }]).not.toEqual(expect.arrayContaining([{ foo: 'bar' }]));
// pass
expect([{ foo: 'bar' }]).not.toEqual(expect.arrayContaining([{}, { foo: 'bar' }]));
```

I think to fix this it would be most straightforward to assert on each "unexpected object" instead:

```ts
unexpectedObjects.forEach((unexpectedObject) => expect(filteredFields.not.toContain(unexpectedObject)));
```